### PR TITLE
Fr fix 150

### PR DESCRIPTION
**Pull request by ludoo0d0a**
Last french translations for 1.5.0.

### CODE REVIEW

File: assets/i18n/fr.json [hunk 1]
```diff
@@ -252,6 +251,7 @@
   "beatnik_title": "Beatnik",
   "beatnik_description": "Langage de programmation ésotérique",
   "beatnik_example": "beau zoos zoo page zoo page camions →  4",
+  "beatnik_download_documentation_url": "https://blog.gcwizard.net/wp-content/uploads/2021/02/beatnik_en.pdf",
   "beatnik_download_documentation_title": "Télécharger un guide de démarrage rapide pour Beatnik",
```
Comment (S-Man42) [inline assets/i18n/fr.json:255]: Please remove the url entry since it not necessary anymore. Below at chef tool, as well

File: assets/i18n/fr.json [hunk 2]
```diff
@@ -2039,7 +2043,7 @@
   "periodictable_comment_cn_1": "En 2010, Ununbium (Uub) a été renommé en Cn",
   "periodictable_comment_fl": "En 2012 Ununquadium (Uuq) a été renommé en Fl",
   "periodictable_comment_lv": "En 2012 Ununhexium (Uuh) a été renommé en Lv",
-  "periodictable_comment_xe": ", Cl, O, N",
+  "periodictable_comment_xe": "Dans des circonstances extrêmes des réactions avec Cl, O, N",
```
Comment (S-Man42) [inline assets/i18n/fr.json:2046]: Please keep the original value because internally the string is used in a concatenation which completes it to a enumeration :)

File: assets/i18n/fr.json [hunk 3]
```diff
@@ -387,41 +387,41 @@
...
+  "changelog_1.4.1": "[chg] Coordonnées Format UTM: Lat Zone (Thanks, Mike)\n[fix] Coordonnées Format MGRS (Thanks, Mike & Thomas)\n[fix] Variable Coordonnées: Projection (Thanks, Frank)\n[fix] Numeral Words: Français (Thanks, 'Don Rodolphos')\n[fix] Progress View (Thanks, Mike)\n[fix] Symbol Tables: Gallifreyan, Flag Alphabet",
+  "changelog_1.4.0": "[nouveau] Open Map\n[nouveau] Vigenère Code Breaker (Thanks, Mike, guballa.de)\n[nouveau] Malbolge (Thanks, Thomas)\n[nouveau] Progress View (Thanks, Mike)\n[nouveau] Chef (Thanks, Thomas)\n[nouveau] Nombre Sequences (Thanks, Thomas)\n[nouveau] Periodic Table: Table View\n[nouveau] Coordonnées Measurement\n[nouveau] Coordonnées: Segment Lines/Bearings\n[nouveau] Cistercian (Thanks, Thomas)\n[nouveau] Maya\n[nouveau] Coord. Format: GeoHex\n[nouveau] Coord. Format: XYZ\n[nouveau] Ellipsoids: Sun, Moon, Planets\n[nouveau] Var. Coordonnées: Own location as input\n[nouveau] Cryptage Disk/Mexican Army Cryptage Disk\n[nouveau] Vigenère: Alphabet values as key\n[nouveau] Symbol Table: D'ni (Nombres) (Thanks, Andreas)\n[chg] Periodic Table: Elements in lists clickable",
```
Comment (S-Man42) [inline assets/i18n/fr.json:405]: Hi :)

I saw you replaced [new] by [nouveau]... Nouveau is too long... The Changelog must be as short as possible, because I just have 500 characters for it for Android releases. So I decided to keep the identifiers fix for the chglog independently from the language:

new
rem
chg
ren
del

File: assets/i18n/fr.json [hunk 4]
```diff
@@ -3000,8 +3005,8 @@
   "symboltables_trafficsigns_germany_title": "Panneaux routiers (Allemagne)",
   "symboltables_ulog_title": "ULOG (Universal langage of the Galaxy)",
   "symboltables_ulog_description": "Du jeu en ligne 'Dark Horizon'",
-  "symboltables_unitology_title": "Unitology",
-  "symboltables_unitology_description": "Écriture de l'Église d'Unitology du jeu de rôle 2D 'Dead Space'",
+  "symboltables_unitology_title": "Unitologie",
```
Comment (S-Man42) [inline assets/i18n/fr.json:3008]: Why these changes? The name of the Role Play Game is Unitology, or is there a French title for it? Please keep the description

File: assets/i18n/fr.json (hunk 1 above)
Comment (ludoo0d0a) [reply assets/i18n/fr.json:255]: @S-Man42 did you remove already all *_url fields ?
I cannot see "beatnik_download_documentation_url" anymore
https://github.com/ludoo0d0a/GCWizard/blob/1.5.0/assets/i18n/fr.json

File: assets/i18n/fr.json (hunk 3 above)
Comment (ludoo0d0a) [reply assets/i18n/fr.json:405]: ok, noted !

File: assets/i18n/fr.json (hunk 1 above)
Comment (S-Man42) [reply assets/i18n/fr.json:255]: Yes. All. Not necessary anymore

File: assets/i18n/fr.json (hunk 4 above)
Comment (ludoo0d0a) [reply assets/i18n/fr.json:3008]: ok, fixed

File: assets/i18n/fr.json (hunk 1 above)
Comment (ludoo0d0a) [reply assets/i18n/fr.json:255]: I hide all _url fields in crowdin (hide = no translation)

Comment (ludoo0d0a) [reply assets/i18n/fr.json:255]: except about_manual_url

Comment (ludoo0d0a) [reply assets/i18n/fr.json:255]: and about_faq_url (to be created in fr)
